app/models/course/discussion/topic.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 class Course::Discussion::Topic < ActiveRecord::Base
   actable
+  class_attribute :global_topic_models
+  self.global_topic_models ||= []
```

Comment:
OK, I will do the update.

But actually I think it's very unlikely we will change class name in development mode, and even we do that probably other things will still break ( associations,component framework, etc ), not sure if it's worth doing that for development though.